docs/Guidelines.md

```diff
@@ -0,0 +1,80 @@
+
+
+
+### Git guidelines
+
+------
+
+#### **GitHub Repository**
+
+------
+
+You can check the code through the following link [maplelabs/opensearch-scaling-manager (github.com)](https://github.com/maplelabs/opensearch-scaling-manager)
+
+
+
+#### **Cloning vs forking**
+
+------
+
+Complete code is present in `master` branch. You can clone or fork from the following branch. To be able to [clone](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) this repository and contribute, you will need to be given write access to the repository. Contact the MapleLabs Development team if you need write access. Permission for write access is reserved with MapleLabs.
+
+To contribute without write access, you can [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) the repository and contribute as needed. If you're planning to leave a fork open for a long time (for example, you're working on a complex set of changes), [sync your fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork) occasionally to avoid merge conflicts.
+
+
+
+#### **Open Issue**
+
+------
+
+- You can find an open issue here if you are intersted in solving [maplelabs/opensearch-scaling-manager (github.com)](https://github.com/maplelabs/opensearch-scaling-manager/issues)
+
+- Search the [open issue](maplelabs/opensearch-scaling-manager (github.com)) to ensure no one else has reported something similar and no one is actively working on similar proposed change.
+- If no one has suggested something similar, open an issue with your suggestion to gather feedback from the community.
+
+
+
+#### **Checks before submitting an issue**
+
+------
+
+- Check whether you have the latest code by refreshing the `master`branch.
+
+- Search the [open_issue](https://github.com/maplelabs/opensearch-scaling-manager/issues) and [closed_issue](https://github.com/maplelabs/opensearch-scaling-manager/issues?q=is%3Aissue+is%3Aclosed) to ensure no one else has reported something similar before.
+- File an [issue ticket](https://github.com/maplelabs/opensearch-scaling-manager/issues/new/choose) by providing all the required information. Failure to provide enough detail may result in slow response from the community.
+- Test with the latest build which is auto generated in github once your raised and it's merged. Build will be under your PR in [issue](https://github.com/maplelabs/opensearch-scaling-manager/actions).
+
+
+
+### Pre-requisites to contribute
+
+------
+
+- AWS credentials to test scaling manager with minimum two nodes in cluster.
+- If AWS credentials are not present, You can make use of our module called simulator to test things. Here is the **[simulator_documentation](https://github.com/maplelabs/opensearch-scaling-manager/blob/master/docs/readme_simulator.md)** on what and how to use.
```

Comment:
Please rephrase this like:
If AWS credentials are not present, contibutors can make use of our module called simulator for testing scaling manger. For more details:  **[simulator_documentation](https://github.com/maplelabs/opensearch-scaling-manager/blob/master/docs/readme_simulator.md)**